msgcntr/messageforums-api/src/bundle/org/sakaiproject/api/app/messagecenter/bundle/Messages.properties

```diff
@@ -678,6 +678,8 @@ pvt_email_footer4_b=To reply to this message click this link to access Messages
 pvt_msgs_replied=Replied messages
 pvt_replied=Already replied
 
+pvt_no_subject=[No subject]
```

Comment:
are there other locations where brackets are used like this?

Reply:
I do see it in a few other places
```
./dav/dav/src/bundle/org/sakaiproject/tool/dav/bundle/Messages.properties:folder = [Folder]
./msgcntr/messageforums-api/src/bundle/org/sakaiproject/api/app/messagecenter/bundle/Messages.properties:stat_display_in_thread=[display in conversation]
./sitestats/sitestats-bundle/src/resources/Messages.properties:user_anonymous=[user identity protected]
```

But those don't seem great, I **think** if we always want brackets around it that would be better to go into the code rather than in the Messages.

So the code would have essentially have
` rrepMsg.setTitle("[" + getResourceBundleString("pvt_no_subject") + "]");`

And this value would just be "No subject". We shouldn't be putting HTML in these properties either but I'm also seeing that right in here too in `pvt_msg_fwd_label`. :/

Reply:
We can do what you prefer. What do we do? :)